### ISSUE TITLE
Create default profile and `CheckList` automatically

### ISSUE THREAD

**Issue by saberduck** [SonarSource/SonarJS#3596]
Currently, whenever we add a new rule we need to maintain list of rule classes (called `Check` ) in `CheckList` and when the rule is part of default profile we need to generate. This creates conflicts when multiple people are implementing rules in parallel.

Goal of this ticket is to create these two lists automatically, so we can avoid merge conflicts when implementing rules. We should keep in mind that profile information is maintained in the `SonarSource/rspec` repository, so we need to find a way how to fetch data from this repository.

**Comment by victor-diez-sonarsource** [comment]
I would probably leave this ticket aside of the hardening until we do the changes we discussed in the technical direction circle, as this topic was mentioned.

**Comment by yassin-kammoun-sonarsource** [comment]
Yeah, it's likely what @alexander-kamushkin-sonarsource (and maybe I) would tackle as a next step from our previous sprint along with rule metadata.

**Comment by Wohops** [comment]
Migrated to Jira. New ticket number: JS-109
